addons/hr_recruitment/migrations/master/pre-update_hr_job.py

```diff
@@ -0,0 +1,6 @@
+from odoo.tools import sql
+from odoo.upgrade import util
+
+def migrate(cr, version):
+    util.remove_field(cr, 'hr.job', 'activities_overdue')
+    util.remove_field(cr, 'hr.job', 'activities_today')
```

Comment:
the upgrade script should be added in a PR to the upgrade repo, targetting version `saas~18.5`
```suggestion
from odoo.upgrade import util

def migrate(cr, version):
    util.remove_field(cr, 'hr.job', 'activities_overdue')
    util.remove_field(cr, 'hr.job', 'activities_today')
```